camel/agents/chat_agent.py

```diff
@@ -134,24 +137,31 @@ def __init__(
 
         self.model_config: BaseConfig
         if self.model_type == ModelType.GPT_4_TURBO_VISION:
```

Comment:
we can also remove`GPT_4_TURBO_VISION` from `ModelType` since the vision model has been integrated as on `GPT_4_TURBO` model

Reply:
Removed the `GPT_4_TURBO_VISION`